projects/vdk-plugins/vdk-kerberos-auth/src/vdk/plugin/kerberos/krb5.conf

```diff
@@ -0,0 +1,15 @@
+[libdefaults]
```

Comment:
is this the default value of a configurable krb5.conf? or is this going to be removed in favour of a configuration that defaults to empty?

Reply:
This krb5.conf file is just a placeholder. In a separate PR I will implement functionality to allow the actual krb5.conf file to be passed in. Then we can eventually remove this conf file entirely.